### PR TITLE
Layout updates, cleaning up after Shapes refactoring

### PR DESCRIPTION
**Pull request by pichfl**
Follow up to #348

### CODE REVIEW

**Review by marcoow — COMMENTED** [review]
seems good to me in general but CI and deployment fails

**Comment by pichfl** [comment]
On it.

File: src/ui/components/ShapeFeature/template.hbs [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
-        id={{this.shapeId}}
-        d="m840.577137 0 862.961153 498.230855-231.96152 401.769145h-512.153907l-862.9611537-498.230855 231.9615247-401.769145z"
-      ></path>
-    </defs>
-    <use fill-rule="evenodd" xlink:href="#{{this.shapeId}}"></use>
-  </svg>
+  <div class="bottom">
+    <div class="bottom-content" style={{this.backgroundStyle}}>
+      {{#if @video}}
+        <video playsinline autoplay muted loop poster={{@background}} class="video" width="x" height="y">
```
Comment (marcoow) [inline src/ui/components/ShapeFeature/template.hbs]: should `width` and `height` receive real values or am I just not getting it?

Comment (pichfl) [reply src/ui/components/ShapeFeature/template.hbs]: They should. I copied the code from the previous component and must have overlooked the invalid values as they are ignored by the browser as it seems. Update incoming.

**Review by marcoow — APPROVED** [review]

**Pull request merged**